src/Discovery/EventBusDiscovery.php

```diff
@@ -14,7 +14,7 @@
 
 final readonly class EventBusDiscovery implements Discovery
 {
-    private const CACHE_PATH = __DIR__ . '/event-bus-discovery.cache.php';
+    private const string CACHE_PATH = __DIR__ . '/event-bus-discovery.cache.php';
```

Comment:
We should add a CSFixer rule for typing constants.

Reply:
I gotchu.